dev/bots/test.dart

```diff
@@ -187,6 +187,9 @@ String get shuffleSeed {
   return _shuffleSeed!;
 }
 
+final bool _isRandomizationOff = const bool.fromEnvironment('TEST_RANDOMIZATION_OFF') ||
```

Comment:
bool.fromEnvironment doesn't check for environment variables, it checks for dart defines. You want to check `Platform.environment['TEST_RANDOMIZATION_OFF']` or something like that (please double check the docs)

Reply:
Platform.environment is checked by next line.

Reply:
But we can add this later if needed.